### DISCUSSION TITLE
breakpoint  within the FB_TestSuite function not work

### DISCUSSION THREAD

**Discussion by cdahinden** [tcunit/TcUnit#216] [Q&A]
Why does a breakpoint not work within the FB_TestSuite function?

**Comment by sagatowski** [comment]
@cdahinden  FB_TestSuite is not a function, it's a function block.
Can you elaborate?

**Software versions**
Which version of the applicable software did you use? (TcUnit library, TwinCAT XAE version, Visual Studio version etc)

**Run environment**
?

**Example code**
?

**Screenshots**
?
